apis/capabilities/v1beta1/backend_types.go

```diff
@@ -168,14 +167,6 @@ func (backend *Backend) SetDefaults(logger logr.Logger) bool {
 		updated = true
 	}
 
-	// 3scale API ignores case of the system name field
```

Comment:
could you please look at comment for product. Thank you

Reply:
Approach to me seems good - however, needed to update this line as well https://github.com/3scale/3scale-operator/blob/master/pkg/helper/openapi_utils.go#L25-L28
Thanks!